src/pdfviewer/PDFDocument.cpp

```diff
@@ -1057,11 +1057,17 @@ void PDFWidget::openAnnotationDialog(const PDFAnnotation *annon)
 	dlg->show();
 }
 
-void PDFWidget::callGetPosFromClick(const QPoint &p){
+void PDFWidget::getPosClick(const QPoint &p){
```

Comment:
as this function was added by a contributor, it is not very nice thing to rename it. The contributors should be able to find their code again unless there is really a compelling reason.
Furthermore the describing comment from the now deleted function should be kept.

Reply:
so it should be either "getPosFromCLick" or "callGetPosFromClick"

Reply:
sorry, I'll change back. For explanation: It's name said that the function calls getPosFromCLick (with upper L). But this is not longer true.

Reply:
so getPosFromClick should be fine.

Reply:
sorry, changed already to call...